src/index.ts

```diff
@@ -38,6 +39,9 @@ async function main() {
         return;
     }
 
+    if (!targetVersionArg.includes(".")) {
+        targetVersionArg = (await getLatestPackageVersion("@comet/admin", semver.coerce(targetVersionArg)?.major)) ?? targetVersionArg;
```

Comment:
Why do we need this fallback to targetVersionArg? When can this happen?

Reply:
Never. This was necessary before https://github.com/vivid-planet/comet-upgrade/pull/51/commits/f37b3454c53d8d080c22b0615895c5fba68f3cbd because I returned null on error.

I removed it in 5c9c1d5eb4b982934bcb282c80df69065e67abb5